pages/apim/3.x/user-guide/publisher/plans/create-plan.adoc

```diff
@@ -0,0 +1,44 @@
+= Create a plan
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_publisherguide_create_plan.html
+:page-folder: apim/user-guide/publisher
+:page-keywords: Gravitee.io, API Platform, API Management, API Gateway, documentation, manual, guide, reference, api, CGU, GCU
+:page-layout: apim3x
+
+== Overview
+
+There are two ways to create a plan in APIM.
+
+=== Create a plan when you create your API
+
+You can create a plan as part of creating an API.
+
+image::apim/3.10/create-api-plan.png[]
+
+Learn more about creating plans using this method in link:/apim/3.x/apim_publisherguide_create_apis.html[Create APIs^].
+
+TIP: Plans you create as part of the API creation process are more limited in scope than plans created using the method below.
+You can use this method to get up and running quickly, then go back and update the plan later if needed.
+
+=== Create or update a plan for an existing API
+
+To create or update a plan:
+
+. Click **APIs** and select your API.
+. Click *Portal > Plans*.
++
+image::apim/3.10/create-plan.png[]
```

Comment:
the format of this screenshot is a little bit disturbing, it does not have the same zoom as previous one making it hard to see the interesting part of this picture: the plan creation form
wdyt ?